module_4/gifts/controller/src/main/java/com/epam/esm/controller/converter/UserRequestViewConverter.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.esm.controller.converter;
+
+import org.modelmapper.AbstractConverter;
+import org.modelmapper.Converter;
+import org.modelmapper.ModelMapper;
+import org.springframework.stereotype.Component;
+
+import com.epam.esm.controller.view.UserRequestView;
+import com.epam.esm.dto.UserDto;
+
+@Component
+public class UserRequestViewConverter {
```

Comment:
@LizavetaSinitsyna UserViewConverter and UserRequestViewConverter the same but with different Generic type. Perhaps you should use abstraction or interface for implementation.